src/components/bible/note-editor.tsx

```
@@ -0,0 +1,163 @@
+            <Loader2 className="animate-spin size-6" />
+        </div>
+    }
+
+    return (
+        <form action="" onSubmit={(e) => {
+            e.preventDefault()
+
+            form.handleSubmit()
+        }} className='border mt-2 sm:p-4 rounded-lg'>
+            <form.Field name="content"
+                children={(field) => (
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)